testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/TestRetryPluginSmokeTest.groovy

```diff
@@ -19,8 +19,10 @@ package org.gradle.smoketests
 import org.gradle.test.fixtures.file.TestFile
 import org.gradle.testkit.runner.BuildResult
 import org.gradle.testkit.runner.TaskOutcome
+import spock.lang.Ignore
 import spock.lang.Issue
 
+@Ignore("Fails since DefaultTestFilter became abstract, will be fixed in next test-retry release")
```

Comment:
❓ How do we keep track of this so that the `@Ignore` will be removed once the fix is in?

Reply:
Good point!

So we have this issue for broken plugins: https://github.com/gradle/gradle/issues/32126. I've added a column to the plugins table `Has ignored smoke test on EAP branch?` so we track also that.